docs/sources/datasources/mssql/query-editor/index.md

```diff
@@ -247,6 +247,9 @@ The resulting table panel:
 
 ## Use time series queries
 
+> **Note:**  
+> Store timestamps in UTC to avoid issues with time shifts in Grafana when using non-UTC timezones.
```

Comment:
https://grafana.com/docs/writers-toolkit/write/shortcodes/#admonition

```suggestion
{{< admonition type="note" >}}
Store timestamps in UTC to avoid issues with time shifts in Grafana when using non-UTC timezones.
{{< /admonition >}}
```